app/views/idv/_acuant_sdk_selfie_capture_form.html.erb

```diff
@@ -37,16 +37,5 @@
     ) %>
   </div>
 
-  <p id='acuant-fallback-text' class='my3 hidden'>
```

Comment:
There's a handful of references to these elements in the corresponding JavaScript which at a glance appear that they won't be degrade gracefully if the elements are no longer present:

https://github.com/18F/identity-idp/blob/0ebeb69bcd667f6595fad0b0a3307b2adb6413a7/app/javascript/app/acuant/document_capture_dom.js#L19-L20

Not sure if it's simple enough to just remove:

https://github.com/18F/identity-idp/blob/0ebeb69bcd667f6595fad0b0a3307b2adb6413a7/app/javascript/packs/acuant_selfie_capture.js#L6

Reply:
nice catch!